system/models/users/login.py

```diff
@@ -0,0 +1,14 @@
+class Login:
+    def __init__(self, user_id: int, user_name: str, password: str, is_admin: bool, error: str):
```

Comment:
I'm not 100% sure on how this implementation will work but the login/register should pass in a User object, initialise it as well as a UserDB (hasn't been merged yet, waiting for a review). These are then used for the various authentication methods.

So I think something like this:
def __init__(self, user: User):
    self._user = user
    self._user_db = UserDB("userDB")